src/braket/circuits/observables.py

```diff
@@ -456,10 +456,7 @@ def __init__(self, observables: List[Observable], display_name: str = "Hamiltoni
 
     @property
     def ascii_symbols(self) -> Tuple[str, ...]:
-        return tuple(
-            "+".join([obs.ascii_symbols[0] for obs in self.summands]).replace("+-", "-")
-            for _ in range(self.qubit_count)
-        )
+        return tuple("Hamiltonian" for _ in range(self.qubit_count))
```

Comment:
should this be using the `display_name` provided in the constructor, rather than a hard-coded string?

Reply:
Absolutely. The correct way to implement this is to just remove the dynamic property method and use the constructor's logic for maintaining the ascii strings-- I've updated the PR

Reply:
nice!